src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In Javascript, combination of function and its lexical environment is called?",
+    Answer: "Closure",
+    Distractor1: "Debouncing",
+    Distractor2: "Throttling",
+    Distractor3: "Scope Chaining",
+    Explanation: "A closure is the combination of a function bundled together (enclosed) with references to its surrounding state (the lexical environment). In other words, a closure gives you access to an outer function's scope from an inner function.",
```

Comment:
This is a direct quote from MDN. We want to avoid plagiarizing 3rd party content. It is always best to create the explanations in your own words. 👍 

```suggestion
    Explanation: "Closures are special types of functions where the inner function will have access to variables in the outer function's scope.",
```